src/js/media-chrome-menu-button.js

```diff
@@ -0,0 +1,147 @@
+import './media-chrome-button.js';
+import './media-chrome-listbox.js';
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import { Window as window, Document as document } from './utils/server-safe-globals.js';
+import { MediaStateReceiverAttributes } from './constants.js';
+
+const template = document.createElement('template');
+template.innerHTML = `
+  <style>
+  :host {
+    display: block;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  media-chrome-button:not(:focus-visible) {
+    outline: none;
+  }
+  </style>
+
+  <media-chrome-button aria-haspopup="listbox" aria-controls="listbox">
```

Comment:
question(non-blocking): since `aria-controls="listbox"` relies on `id="listbox"` on the default slotted listbox, won't this break for any externally-slotted listbox? Haven't tested yet in your `media-chrome-menu-button` example, but I believe the captions use case should be sufficient to test this?

Reply:
yeah, that makes sense. Not exactly sure how to handle this, since we probably don't want to add an ID to a slotted component if it doesn't already have one?

Unfortunately, the new upcoming additions to aria that help in this case won't be available for a while (also, I'm not sure if there's a settled solution quite yet or that there's multiple solutions).

Reply:
https://www.w3.org/WAI/ARIA/apg/patterns/menubutton/ says that aria-controls is optional.
So, I think for the MVP there are two options:
- not use it at all
- use it but when a listbox is slotted, remove aria-controls from the button.

Reply:
Based on this article, https://heydonworks.com/article/aria-controls-is-poop/ I say we ignore that attribute altogether.